api/src/main/java/org/pmiops/workbench/profile/ProfileMapper.java

```diff
@@ -46,6 +46,8 @@ Profile toModel(
       List<UserTierEligibility> tierEligibilities,
       ProfileAccessModules accessModules);
 
+  // temporary for deployment safety - will be removed after one release cycle.
+  @Mapping(source = "duccBypassTime", target = "dataUseAgreementBypassTime")
```

Comment:
I do not understand this. Should the field on DbAdminTableUser be renamed too, or would that cause a problem?

Reply:
This is not strictly necessary but it may give us a small amount of freedom of action (if I'm thinking about this correctly).

Before:
AdminTableUser and DbAdminTableUser both had a `dataUseAgreementBypassTime` so the mapper connected them automatically.

Now:
I have added `duccBypassTime` to both, which the mapper will also connect automatically.
I need to retain the entity `dataUseAgreementBypassTime` for one release cycle.  If I change its mapping to the DB `duccBypassTime`, I can then remove the DB `dataUseAgreementBypassTime` whenever, because it's no longer connected to anything.

So... could I remove `DbAdminTableUser.dataUseAgreementBypassTime` now?  Probably.  But I think I will leave it because I'm not 100% sure.